tools/model.go

```diff
@@ -285,7 +283,7 @@ func getProjection(rm *RasModel, fn string, wg *sync.WaitGroup) {
 
 	sourceSpRef := gdal.CreateSpatialReference(line)
 	if err := sourceSpRef.Validate(); err != nil {
-		fmt.Println(fmt.Sprintf("%s is not a valid projection file.\n", fn))
+		fmt.Println(fmt.Sprintf("%s is not a valid projection file.", fn))
```

Comment:
Should this be log as well?

Reply:
I have only switched the errors to logs (not the general print statements) so that they are reported as stderr in the docker logs. The purpose of this was to identify errors in the container logs.

There will be a separate PR soon to log and report errors more elegantly.